cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,11 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i '/,admin,system:masters$/d' "${basic_auth_csv}"
+      # The following is for the legacy form of the password line.
+      sed -i '/,admin$/d' "${basic_auth_csv}"
```

Comment:
If this were just GKE I think that would be the best strategy. However as you pointed out in GCE the client may have added their own accounts that we should not wipe out.